source/chapters/user_interface.rst

```diff
@@ -913,15 +922,15 @@ on the left end of the crossfader and vice versa. This is also known as
 Headphone and Master Mix Controls
 ---------------------------------
 
-.. figure:: ../_static/2.2/master-head-controls.png
+.. figure:: ../_static/2.3/headphone-and-master-mix.png
```

Comment:
Here and everywhere you encounter it:
please change `Master` to `Main`

Reply:
These references raise warnings during Sphinx build. Will they be the exception?

![image](https://user-images.githubusercontent.com/28389950/111478697-5d210080-8741-11eb-914b-2002862e7a63.png)

Reply:
Sorry for not beong clear emough: I referred to all occurrences of MASTER labrls in the mixer and maybe also for Fx units (if those ate included in this PR, too)
main-sync has no GUI label yet, it's - if at all- in some tooltips